installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -75,7 +75,10 @@
             "API": "build-pack-api",
             "LAMBDA": "build-pack-lambda",
             "WEBSITE": "build-pack-website"
-        }
+        },
+        "LAMBDA_SERVICES": ["cloud-logs-streamer", "deployments-event-handler", "environment-event-handler"],
```

Comment:
Can we add this section under a new key - VPC_SERVICES? `LAMBDA_SERVICES`/`API_SERVICES` are too generic & doesn't explain the usage.

Also, I see that they are under JENKINS key, which looks incorrect. Move them under JAZZ.  Proposing `JAZZ.VPC_SERVICES.FUNCTIONS` & `JAZZ.VPC_SERVICES.APIS`. Thoughts?